<ui version="4.0" >
 <author>Stefan Duffner</author>
 <class>TransitionPropertiesDlg</class>
 <widget class="QDialog" name="TransitionPropertiesDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Transition Properties</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="leftMargin" >
    <number>11</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>11</number>
   </property>
   <property name="bottomMargin" >
    <number>11</number>
   </property>
   <item>
    <widget class="Q3ButtonGroup" name="bg_type" >
     <property name="title" >
      <string>Condition Type</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>11</number>
      </property>
      <property name="topMargin" >
       <number>11</number>
      </property>
      <property name="rightMargin" >
       <number>11</number>
      </property>
      <property name="bottomMargin" >
       <number>11</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_bin" >
        <property name="text" >
         <string>Binary</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_ascii" >
        <property name="text" >
         <string>ASCII</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_text" >
        <property name="text" >
         <string>Free Text</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <item row="4" column="0" >
      <widget class="QLabel" name="TextLabel7" >
       <property name="text" >
        <string>Output:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QCheckBox" name="cb_invert" >
       <property name="text" >
        <string>Invert condition</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="le_input" />
     </item>
     <item row="3" column="1" >
      <widget class="QCheckBox" name="cb_default" >
       <property name="text" >
        <string>Default transition</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QCheckBox" name="cb_any" >
       <property name="text" >
        <string>Any input</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QLineEdit" name="le_output" />
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="TextLabel6" >
       <property name="text" >
        <string>Input:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="TextLabel3" >
     <property name="text" >
      <string>Description:</string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Q3TextEdit" name="te_description" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="PushButton19" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushButton20" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_input</tabstop>
  <tabstop>le_output</tabstop>
  <tabstop>rb_bin</tabstop>
  <tabstop>rb_ascii</tabstop>
  <tabstop>PushButton19</tabstop>
  <tabstop>PushButton20</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>PushButton19</sender>
   <signal>clicked()</signal>
   <receiver>TransitionPropertiesDlg</receiver>
   <slot>validate()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PushButton20</sender>
   <signal>clicked()</signal>
   <receiver>TransitionPropertiesDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_bin</sender>
   <signal>clicked()</signal>
   <receiver>TransitionPropertiesDlg</receiver>
   <slot>binaryClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_ascii</sender>
   <signal>clicked()</signal>
   <receiver>TransitionPropertiesDlg</receiver>
   <slot>asciiClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_any</sender>
   <signal>clicked()</signal>
   <receiver>TransitionPropertiesDlg</receiver>
   <slot>anyClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_default</sender>
   <signal>clicked()</signal>
   <receiver>TransitionPropertiesDlg</receiver>
   <slot>defaultClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_invert</sender>
   <signal>clicked()</signal>
   <receiver>TransitionPropertiesDlg</receiver>
   <slot>invertClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_text</sender>
   <signal>clicked()</signal>
   <receiver>TransitionPropertiesDlg</receiver>
   <slot>freeTextClicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
